installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc

```diff
@@ -21,23 +21,19 @@ link:https://access.redhat.com/articles/4207611[guidelines for deploying {produc
 
 == Prerequisites
 
-* xref:../../installing/installing-mirroring-installation-images.adoc#installing-mirroring-installation-images[Create a registry on your mirror host] and obtain the `imageContentSources` data for your version of {product-title}.
-* Move or remove any existing installation files, before you begin the installation process. This ensures that the required installation files are created and updated during the installation process.
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing-mirroring-installation-images.adoc#installing-mirroring-installation-images[created a registry on your mirror host] and obtained the `imageContentSources` data for your version of {product-title}.
+* You must move or remove any existing installation files, before you begin the installation process. This ensures that the required installation files are created and updated during the installation process.
 +
 [IMPORTANT]
 ====
 Ensure that installation steps are done from a machine with access to the installation media.
 ====
-* Provision
-xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage]
-using NFS for your cluster. To deploy a private image registry, your storage
+* You provisioned xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] using NFS for your cluster. To deploy a private image registry, your storage
```

Comment:
Tiny nit. Above you put [[persistent storage using NFS]](https://github.com/openshift/openshift-docs/pull/33681/files#diff-585ba1ab8d18dd09406b2fab278303b7f71c951826834e70f8422b56a2d760f6R27) all in the brackets.

Reply:
Good catch. I have now updated this and updated the cross-reference to link to [Persistent storage using NFS](https://deploy-preview-33681--osdocs.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-nfs.html#persistent-storage-nfs), in line with the other prerequisites that relate to persistent storage on NFS specifically.